copier.yml

```diff
@@ -4,16 +4,16 @@ _exclude:
 _subdirectory: template
 project_title:
     type: str
-    placeholder: "NREL's Python Package"
+    placeholder: "NREL's Modern Data Standards"
     help: What is your project title (used in the documentation)?
 project_name:
     type: str
-    placeholder: "NREL-Python-package"
-    help: What is your project name?
+    placeholder: "NREL-Modern-Data"
+    help: What is your project name? Can use hyphens, but don't use spaces or special characters.
```

Comment:
So is this supposed to be your GitHub repo name? If so, would it be worth mentioning that?

Reply:
I don't have a clear opinion on how to handle this. I didn't want to force just the valid package names, but I'm having trouble to explain it in a few words. This is the case of `PyMFI` vs `mfi`. Do you have an opinion?